### PR TITLE
Updated documentation and example

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updates documentation and example

Summary of changes:
- capitalized Snap
- updated docs
- updated example

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,84 +87,72 @@ User need to provide following parameters in configuration for collector:
 See example task manifest in [examples/tasks/] (https://github.com/intelsdi-x/snap-plugin-collector-glance/blob/master/examples/tasks/).
 
 ### Examples
-Example running snap-plugin-collector-glance plugin and writing data to a file.
+Example of running Snap glance collector and writing data to file.
 
-Make sure that your `$SNAP_PATH` is set, if not:
+Download an [example Snap global config](https://github.com/intelsdi-x/snap-plugin-collector-glance/blob/master/examples/cfg/) file.
```
Comment (andrzej-k) [inline README.md]: Isn't the file name (cfg.json) missing in the link?

File: README.md [hunk 2]
```diff
@@ -84,84 +87,72 @@ User need to provide following parameters in configuration for collector:
...
-Load file plugin for publishing:
+See all available metrics:
+
 ```
-$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PATH/plugin/snap-publisher-file
+$ snapctl metric list
 ```
-See available metrics for your system:
+
+Download an [example task file](https://github.com/intelsdi-x/snap-plugin-collector-glance/blob/master/examples/tasks/) and load it:
```
Comment (andrzej-k) [inline README.md]: Link to file?

File: README.md [hunk 3]
```diff
@@ -27,30 +27,33 @@ It can run locally on the host, or in proxy mode (communicating with the host vi
...
 ```
-This builds the plugin in `/build/rootfs`
+This builds the plugin in `./build/`
 
 ### Configuration and Usage
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started).
-* Create Global Config, see description in [snap's Global Config] (https://github.com/intelsdi-x/snap-plugin-collector-glance/blob/master/README.md#snaps-global-config).
-* Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-glance/blob/master/README.md#examples).
+* Set up the [Snap framework](https://github.com/intelsdi-x/snap#getting-started).
+* Create Global Config, see description in [Snap's Global Config] (https://github.com/intelsdi-x/snap-plugin-collector-glance#snaps-global-config).
```
Comment (kindermoumoute) [inline README.md]: Link to `#snaps-global-config` to make it works across different branch/fork?

File: README.md [hunk 4]
```diff
@@ -27,30 +27,33 @@ It can run locally on the host, or in proxy mode (communicating with the host vi
...
+* Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-glance#examples).
```
Comment (kindermoumoute) [inline README.md]: Same, maybe change the URL to `#examples`?

File: README.md (hunk 3 above)
Comment (marcin-krolik) [inline README.md]: done

File: README.md (hunk 4 above)
Comment (marcin-krolik) [inline README.md]: done

**Pull request merged**